src/assets/TokensController.ts

```diff
@@ -268,6 +295,68 @@ export class TokensController extends BaseController<
     }
   }
 
+  /**
+   * Adds isERC721 field to token object
+   * (Called when a user attempts to add tokens that were previously added which do not yet had isERC721 field)
+   *
+   * @param {string} tokenAddress - The contract address of the token requiring the isERC721 field added.
+   * @returns {Promise<object>} The new token object with the added isERC721 field.
```

Comment:
Nit: Usually in TypeScript repos we omit the type, because it's already documented in the type signature.

```suggestion
   * @param tokenAddress - The contract address of the token requiring the isERC721 field added.
   * @returns The new token object with the added isERC721 field.
```

Reply:
addressed!